worldengine/cli/main.py

```diff
@@ -313,6 +314,14 @@ def main():
                              "If not provided, the default values will be used. \n" +
                             "[default = .059/.222/.493/.764/.927/.986/.998]",
                             metavar="#/#/#/#/#/#/#")
+    g_generate.add_argument('-gv', '--gamma-value', dest='gv', type=float,
```

Comment:
That's because this is the same formula.

Y = X^gamma where both X and Y are in the range of 0-1.